package.json

```diff
@@ -16,5 +16,6 @@
     "typescript": "3.7.2"
   },
   "dependencies": {
+    "jq-web": "0.5.1"
```

Comment:
You may not need this , you could use `get` function from lodash.

Reply:
The scheme of index permission is like: 

~~~
index_permissions: [{
        index_patterns: [""],
        fls: [""],
        masked_fields: [""],
        allowed_actions: [""]
}]
~~~

We need to take all index_patterns from the array and then flatten it so not achievable by single `get`.